src/support/components/FreeAccountSupportOverlay.tsx

```diff
@@ -32,6 +35,15 @@ const FreeAccountSupportOverlay: FC<OwnProps> = () => {
   const {onClose} = useContext(OverlayContext)
   const isCredit250ExperienceActive = useSelector(shouldGetCredit250Experience)
 
+  const dispatch = useDispatch()
+
+  const openFeedbackOverlay = (event): void => {
+    event.preventDefault()
```

Comment:
I don't think the `event.preventDefault()` is necessary here. What happens when you remove it?

Also, do we want an `event` here?

Reply:
@blegesse-git cool, looks like the move is to leave the `eventPrevent` :) (nice answer too heh)

do we need an app event like `event('helpBar.freeAccount.supportOverlay.shown')`

Reply:
haha! yes. i didn't realize i answered your question with a question. 
and i thought about that too. I think adding an event metrics there could be helpful. But that means adding event metrics for both payg and free account overlays  inside `TreeNav` component as well. Which is a-okay with me. I'll get the fix up soon